src/mgxs_header.F90

```diff
@@ -970,11 +964,6 @@ subroutine mgxs_print(this, unit_)
 
       ! Basic nuclide information
```

Comment:
We can get rid of these mgxs_print routines too.  You don't need to do it, I can cover it when I do a similar re-factor of MGXS.  But I might need you to remind me!